src/sass/_shame.scss

```diff
@@ -69,18 +69,6 @@ body .row {
   }
 }
 
-.review {
```

Comment:
So yes. Deleting that line removed the blue color from the :firstchild in both HCA and Edu. It looks like I added the inverse fix to a slightly more specific place that is only affecting Edu. It SHOULD get fixed in both places. @goldenmeanie can take care of that in the branch he's currently working on.

Sorry I jumped the gun. The interim state is fine in the short term.